schemas/manifest.schema.json

```
@@ -0,0 +1,473 @@
+            "type": "object",
+            "required": ["en"],
+            "additionalProperties": false,
+            "patternProperties": {
+                "^[a-z]{2}$": {
+                    "type": "string"
+                }
+            }
+        },
+        "byte_size": {
+            "type": "string",
+            "pattern": "^[0-9]*(\\.[0-9]*)?[kKmMgG]$"
+        },
+        "boolean_or_not_relevant": {
+            "anyOf": [
+                {
+                    "type": "boolean"
+                },
+                {
+                    "type": "string",
+                    "const": "not_relevant"
+                }
+            ]
+        },
+        "string_or_array_of": {
+            "anyOf": [
+                {
+                    "type": "string"
+                },
+                {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                }
+            ]
+        }
+    },
+
+    "properties": {
+        "packaging_format": {
+            "description": "Packaging format / version",
+            "type": "integer"
+        },
+        "id": {
+            "description": "App ID",
+            "type": "string"
+        },
+        "name": {
+            "description": "App Name",
+            "type": "string"
+        },
+        "version": {
+            "description": "App and package version",
+            "type": "string",
+            "pattern": "^[0-9\\.]*~ynh[0-9]*"
+        },
+        "description": {"$ref": "#/$defs/translated_string"},
+        "maintainers": {
+            "description": "List of maintainers",
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+        },
+        "upstream": {
+            "type": "object",
+            "required": ["license"],
+            "additionalProperties": false,
+            "properties": {
+                "license": {"type": "string"},
+                "website": {"type": "string", "format": "uri-reference"},
+                "demo": {"type": "string", "format": "uri-reference"},
+                "admindoc": {"type": "string", "format": "uri-reference"},
+                "userdoc": {"type": "string", "format": "uri-reference"},
+                "code": {"type": "string", "format": "uri-reference"},
+                "fund": {"type": "string", "format": "uri-reference"},
+                "cpe": {
+                    "type": "string",
+                    "pattern": "^cpe:[0-9\\.]*:[a-z]*:[a-zA-Z0-9_]*:[a-z]*$"
+                }
+            }
+        },
+        "integration": {
+            "type": "object",
+            "required": ["yunohost", "architectures", "multi_instance", "ldap", "sso"],
+            "additionalProperties": false,
+            "properties": {
+                "yunohost": {
+                    "type": "string",
+                    "pattern": "^>= ?[0-9]*\\.[0-9]*\\.[0-9]*"
```

```suggestion
                    "pattern": "^>= ?[0-9]*\\.[0-9]*\\.[0-9]*"
```

Ideally we should allow for `>= 11.x` or even `>= 11`, or not if too annoying to implement :sweat_smile:

---

```suggestion
                    "pattern": "^>= ?[0-9]*(?:\\.[0-9]*)*"
```
(untested)

---

I found even better :
`^>= ?[0-9]{1,2}(\.[0-9]{1,2}){0,2}$`